mesos-docker/run/run.sh

```diff
@@ -230,6 +238,23 @@ function start_slaves {
       HADOOP_VOLUME=
     fi
 
+    resources_cfg=$(get_field_value_from_slave_cfg_at_index resources $SLAVES_CONFIG_FILE $(($i-1)))
+    attributes_cfg=$(get_field_value_from_slave_cfg_at_index attributes $SLAVES_CONFIG_FILE $(($i-1)))
```

Comment:
Those 2 lines fail when `--slaves-cfg-file` is not specified. So there is still a need to run them in a `if` block.